netlink/conntracker.go

```diff
@@ -0,0 +1,266 @@
+// +build linux
+
+package netlink
+
+import (
+	"context"
+	golog "log"
+	"net"
+	"os"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	ct "github.com/florianl/go-conntrack"
+)
+
+// Conntracker is a wrapper around go-conntracker that keeps a record of all connections in user space
+type Conntracker interface {
+	GetConntrackEntryForConn(ip string, port uint16) *IPTranslation
```

Comment:
nit: this is redundant with the interface name. Just `GetTranslationForConn` should be fine.